src/tiled/abstractobjecttool.cpp

```diff
@@ -115,8 +116,7 @@ void AbstractObjectTool::mouseMoved(const QPointF &pos,
 void AbstractObjectTool::mousePressed(QGraphicsSceneMouseEvent *event)
 {
     if (event->button() == Qt::RightButton) {
-        showContextMenu(topMostObjectItemAt(event->scenePos()),
-                        event->screenPos());
+        showContextMenu(event->scenePos(), event->screenPos());
```

Comment:
Return what back? The menu? That's not possible because the menu is destroyed before the function returns. If we need it, we should have a `createContextMenu`, but I think we don't need it.